e2e/test_operator/operator_test.go

```diff
@@ -1747,4 +1747,67 @@ var _ = Describe("Operator", Label("e2e", "pr"), func() {
 			Expect(cluster.Status.DatabaseConfiguration.RoleCounts.Proxies).NotTo(BeZero())
 		})
 	})
+
+	When("running with tester processes", func() {
+		BeforeEach(func() {
+			// We will be restarting the CC, so we can ignore this check.
+			availabilityCheck = false
+			spec := fdbCluster.GetCluster().Spec.DeepCopy()
+			spec.ProcessCounts.Test = 1
+			fmt.Println("Adding 1 tester process to the cluster")
+			fdbCluster.UpdateClusterSpecWithSpec(spec)
+			Expect(fdbCluster.WaitForReconciliation()).NotTo(HaveOccurred())
+		})
+
+		It("when the tester process fails", func() {
```

Comment:
We should add a test case here for the case where the tester is able to connect to the CC but the CC is not able to connect to the CC to better understand the behaviour in this scenario.